services/bettersocial/chat/updateBetterChannelMembers.js

```diff
@@ -29,6 +29,8 @@ const updateBetterChannelMembers = async (
   if (!channel) throw new Error('Channel is required');
 
   const {members} = channel;
+  console.log('channel', channel);
```

Comment:
ini log sengaja di tinggalin atau kelupaan blm di hapus ya mas ?

Reply:
oh iya ini saya remove dulu mas. thank you.